editor_constants.py

```diff
@@ -33,5 +36,36 @@
         slant=FONT_TAG_PREVIEW_FONT_SLANT,
         )
     )
+
+WORKING_DIR = Path.cwd()
+if b_e_c.IS_WIN:
+    SETTINGS_DIR = WORKING_DIR
+else:
+    SETTINGS_DIR = Path(Path.home(), ".local", "share", "mek", "mozzarilla")
+
+MOZZLIB_DIR = Path(__file__).parent
+
+MOZZ_ICON_PATH = Path(MOZZLIB_DIR, "mozzarilla.ico")
+if not MOZZ_ICON_PATH.is_file():
+    MOZZ_ICON_PATH = Path(MOZZLIB_DIR, "icons", "mozzarilla.ico")
+if not MOZZ_ICON_PATH.is_file():
+    MOZZ_ICON_PATH = Path(WORKING_DIR, "mozzarilla.ico")
+if not MOZZ_ICON_PATH.is_file():
+    MOZZ_ICON_PATH = Path(WORKING_DIR, "icons", "mozzarilla.ico")
+if not MOZZ_ICON_PATH.is_file():
+    MOZZ_ICON_PATH = ""
+
+MOZZ_BITMAP_PATH = Path(MOZZLIB_DIR, "mozzarilla.png")
+if not MOZZ_BITMAP_PATH.is_file():
+    MOZZ_BITMAP_PATH = Path(MOZZLIB_DIR, "icons", "mozzarilla.png")
+if not MOZZ_BITMAP_PATH.is_file():
+    MOZZ_BITMAP_PATH = Path(WORKING_DIR, "mozzarilla.png")
+if not MOZZ_BITMAP_PATH.is_file():
+    MOZZ_BITMAP_PATH = Path(WORKING_DIR, "icons", "mozzarilla.png")
+if not MOZZ_BITMAP_PATH.is_file():
+    MOZZ_BITMAP_PATH = ""

```

Comment:
If we consistently store the icons in the library folders then this is not needed.